Add docs for EDP 1.4.0 (#2637)
TykTechnologies/tyk-docs: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal/configuration.md

**Type:** `string` <br/>
**Options:**
- `fs` to use file system storage type;

> - `db` to use the portal's main database. In the `db` is selected as a storage type, the portal application will create appropriate structure in the database that is used to store the portal's metadata during [the bootstrap process]({{< ref "/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal/bootstrapping-portal.md" >}});

Reviewer: ```suggestion
- `db` to use the portal's main database. If the `db` is selected as a storage type, the portal application will create appropriate structure in the database that 
```
